python/dftd4/interface.py

```diff
@@ -276,14 +276,15 @@ def __init__(
         charge: Optional[float] = None,
         lattice: Optional[np.ndarray] = None,
         periodic: Optional[np.ndarray] = None,
+        model: Optional[str] = None,
```

Comment:
Please make this mandatory, but provide a proper default. Note that `"d4+something"` or other model names currently just silently pass.
```suggestion
        model: str = "d4",
```